### ISSUE TITLE
inference of reverse binding from MF has_direct_input type annotations

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#793]
Increasingly for GO oannotation
activity of  x has_input  y  replaces   x binds y
However, for our binding annotations we check that they are reciprocated 
We can annotate  x binds y and we will also generate y binds x.

Do we currently have anything in place  to provide 
y binds x if  activity of  x has_input  y ?
I can't remember.....

**Comment by kimrutherford** [comment]
> Do we currently have anything in place to provide
> y binds x if activity of x has_input y ?

I don't think so.  But I'm not sure I completely understand the question.

**Comment by ValWood** [comment]
So here

cut12 acts as a "signalling adaptor" for  plo1

<img width="811" alt="Screenshot 2020-11-20 at 11 39 19" src="https://user-images.githubusercontent.com/7359272/99796254-13266600-2b25-11eb-9fcb-7f1849781008.png">


this means that plo1 binds cut12

Here I have curated this
plo1
binds cut12 part of positive regulation of G2/M transition of mitotic cell cycle | IPI | Grallert A et al. (2013)
(NOTE TO SELF, change to PR:000050399)

So I  cant remember if we already discussed an implemented to instantiate the 'opposite'  (i.e plo1 binds cut12) based on the "activity". I don't think we did.

An added complication is that the binding annotations DO get reciprocated automatically

and if 
plo1
binds cut12 part of positive regulation of G2/M transition of mitotic cell cycle | IPI | Grallert A et al. (2013)

is reciprocated, this is redundant with the "signalling adaptor activity" annotation.

So, I'm not sure how we should proceed here. It might be easier to discuss possible ways we could  work this on a call. It isn't urgent though.
